src/south/OPCUA/OPCUA_DA/OPCUA_DA.class.js

```diff
@@ -147,7 +156,7 @@ class OPCUA_DA extends ProtocolHandler {
       return
     }
 
-    this.subscription = Opcua.ClientSubscription.create(this.session, {
+    this.subscription = ClientSubscription.create(this.session, {
```

Comment:
i am not completely sure why this code is commented and why we would make change to commented code. Should we plainly remove this piece of code?

Reply:
This subscription was developed by Szabi a while ago, but the management of subscription associated to the points needed more conception. So this piece of code has been commented for future used.

Reply:
i find that usually very unpractical to leave commented code, we should open a branch with this rather than cluttering code with comments (and we don't even know why this is commented)